src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const obj = {
+    result: 0,
+
+    add: (value) => {
```

Comment:
Will be better if you fix it everywhere
<img width="1131" alt="image" src="https://github.com/mate-academy/js_advanced_calculator/assets/64750958/6f873207-dab6-4d1e-bde0-c76523f1b6f6">
```suggestion
    add(value) {
```